### ISSUE TITLE
screen roll change

### ISSUE THREAD

**Issue by andrew-davie** [stella-emu/stella#533]
Welp.... I've just spent MANY hours hunting this one down.
It seems there is a behaviour change in how Stella interprets/presents screen roll.
The symptom I see is the screen start about 1/4 way down the screen and slowly move back up to the correct position. I thought I was doing something weird in my current code, but it turns out that this is actually a difference between the current released version and the beta version.
In particular, I can load the same ROM file in old stella - works fine, locked screen - and in the new stella - screen does the 'roll' up to correct position.
Normally I wouldn't complain too much about this, but for example the release binary of Boulder Dash exhibits this behaviour, and TVs definitely do not show this happening.
In short, the VBLANK of BD and my current game are a bit 'naughty' in that they do not have the requisite 37 lines of "screen off". I could probably pare this down to an example binary if really required - but first I thought I'd ask... why was this changed, and what are the perceived advantages?

**Comment by andrew-davie** [comment]
Here are screen videos of old (release) and new (beta). You'll need to unzip the attached file to view. Look at the transition to the actual game screen. In the new beta it 'rolls' and in the released version it does not. Behaviour change.

[screengrabs.zip](https://github.com/stella-emu/stella/files/3574290/screengrabs.zip)

**Comment by thrust26** [comment]
I have to check what has changed (I only remember vaguely), but a too short VBlank **can** lead to rolling CRTs. It depends on the brightness set and what is displayed when VBlank is off. If the display is all black and the setting is dark enough, then the TV can interpret this a VBlank enabled.

But the behavior is different between different CRTs.

**Comment by andrew-davie** [comment]
I only noticed this after doing a recent pull on the code. I would be guessing it's a difference that happened recently, otherwise I expect I would have noticed it in the previous code I was using...?

**Comment by thrust26** [comment]
Except  09bd0f8 I cannot see any commit during the last weeks which might have changed behavior.

**Comment by andrew-davie** [comment]
This is not present in the very very latest code, and can probably be closed.
Having said that, it would have been nice to know the cause and fix.

**Comment by thrust26** [comment]
That's the inevitable risk when using WIPs. 

If you are really interested, you can always try to bisect the commits. :smile:
